natgeo/mortar
# [SYTO-915] Dumb fix for not having borders on video elements.

Generic styles in docs apply border to any `td, th`.
This fix overrides with no-border for td,th elements within video component.

To test it:
- disable flash in browser (so HTML video version gets loaded)
- scroll down in http://192.168.0.8.xip.io:3000/components/ until video
- check that video controls at bottom look good without border (as you may see on same check on current qa site)
- have a beer
